components/docs-chef-io/content/automate/ha_backup_restore_aws_efs.md

```diff
@@ -27,7 +27,7 @@ gh_repo = "automate"
 
 A shared file system is always required to create **OpenSearch** snapshots. To register the snapshot repository using OpenSearch, it is necessary to mount the same shared filesystem to the exact location on all master and data nodes. Register the location in the `path.repo` setting on all master and data nodes.
 
-### Setting up the backup configuration
+### Setup your Backup Configuration
```

Comment:
"Setup" is a noun, this should be a verb. So use "Set up". 

Also, headings should be sentence case. Use the [sentence case converter](https://titlecaseconverter.com/sentence-case/) if you're not sure. I've left this comment twice before. If you're updating headings, make them sentence case.

Reply:
I have fixed the headings according to the suggested converter. 
Take a look once.